dojo/api_v2/serializers.py

```diff
@@ -2454,9 +2462,18 @@ def set_context(
         context = dict(data)
         # update some vars
         context["scan"] = data.get("file", None)
-        context["environment"] = Development_Environment.objects.get(
-            name=data.get("environment", "Development"),
-        )
+
```

Comment:
Now that there is more going on here, should we make this a function so that we dont have copy past code? For maintainability it would be good to have only one place where this identical code resides.

Reply:
This was a fairly large lift with regard to changes in that file so I will request re-reviews.